ts/features/itwallet/identification/hooks/useNoCieBottomSheet.tsx

```diff
@@ -0,0 +1,95 @@
+import { StyleSheet, View } from "react-native";
+import {
+  FooterActions,
+  Pictogram,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n";
+import { useIOBottomSheetModal } from "../../../../utils/hooks/bottomSheet";
+import IOMarkdown from "../../../../components/IOMarkdown";
+import { ItwEidIssuanceMachineContext } from "../../machine/provider";
+
+const noCieContentKeys = [
+  "features.itWallet.identification.l3.mode.bottomSheet.noCie.content.firstAttention",
+  "features.itWallet.identification.l3.mode.bottomSheet.noCie.content.secondAttention",
+  "features.itWallet.identification.l3.mode.bottomSheet.noCie.content.thirdAttention"
+] as const;
+
+/**
+ * Hook to manage the no CIE bottom sheet
+ */
+export const useNoCieBottomSheet = () => {
+  const machineRef = ItwEidIssuanceMachineContext.useActorRef();
+  const navigateToL2Identification = () => {
+    machineRef.send({ type: "go-to-l2-identification" });
+    noCieBottomSheet.dismiss();
+  };
+  const noCieBottomSheet = useIOBottomSheetModal({
+    title: I18n.t(
+      "features.itWallet.identification.l3.mode.bottomSheet.noCie.title"
+    ),
+    component: (
+      <>
+        <IOMarkdown
+          content={I18n.t(
+            "features.itWallet.identification.l3.mode.bottomSheet.noCie.description"
+          )}
+        />
+        <VSpacer size={8} />
+        <View>
+          {noCieContentKeys.map((key, index) => (
+            <View key={index} style={styles.row}>
+              <View style={styles.iconContainer}>
+                <Pictogram name="attention" size={64} />
+              </View>
+              <View style={styles.textContainer}>
+                <IOMarkdown content={I18n.t(key)} />
+              </View>
+            </View>
+          ))}
+        </View>
+        <FooterActions
```

Comment:
We should not use `FooterActions` inside a bottom sheet, as it has different margins and spacing. We need to use `IOButton`